vllm/model_executor/layers/activation.py

```diff
@@ -27,13 +30,10 @@ class SiluAndMul(nn.Module):
     The function computes x -> silu(x[:d]) * x[d:] where d = x.shape[1] // 2.
     """
 
-    def __init__(self):
-        super().__init__()
-
     def forward(
-        self,
-        x: torch.Tensor,        # (num_tokens, 2 * d)
-    ) -> torch.Tensor:          # (num_tokens, d)
+            self,
+            x: torch.Tensor,  # (num_tokens, 2 * d)
+    ) -> torch.Tensor:  # (num_tokens, d)
```

Comment:
Is this code not allowed by our formatter?
```suggestion
        self,
        x: torch.Tensor,  # (num_tokens, 2 * d)
    ) -> torch.Tensor:  # (num_tokens, d)
```

Reply:
Yes. Seems like yapf outputs weird results for inline comments. Move these comments to docstring